sdk/v2/core/jobs_test.go

```diff
@@ -0,0 +1,240 @@
+package core
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestJobSpecMarshalJSON(t *testing.T) {
+	requireAPIVersionAndType(t, JobSpec{}, "JobSpec")
+}
+
+func TestJobStatusMarshalJSON(t *testing.T) {
+	requireAPIVersionAndType(t, JobStatus{}, "JobStatus")
+}
+
+func TestJobsClientCreate(t *testing.T) {
+	const testEventID = "12345"
+	const testJobName = "Italian"
```

Comment:
🇮🇹 🚗 💨

Reply:
You can tell I had some fun with naming test fixtures.